src/api/LiikuntaApolloClient.ts

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment, @typescript-eslint/no-explicit-any */
```

Comment:
Apollo's types are not accurate which is why type checking results have been ignored in this file.